app/src/main/java/io/mochahub/powermeter/models/Exercise.kt

```diff
@@ -1,21 +1,7 @@
 package io.mochahub.powermeter.models
 
-import io.mochahub.powermeter.data.Exercise as ExerciseDataModel
-
 data class Exercise(
     val name: String,
     val personalRecord: Double,
     val muscleGroup: String // TODO: Make MuscleGroup data/enum class?
-) {
-    override fun toString(): String {
-        return "Exercise(name='$name', personalRecord='$personalRecord' muscleGroup='$muscleGroup')"
-    }
-}
-
-/**
- *  Not entirely sure how I feel about the db model and the existing model but this extension function
- *  is a useful mapper from one to the other
- */
-fun Exercise.toDataModel(): ExerciseDataModel {
-    return ExerciseDataModel(name = this.name, personalRecord = this.personalRecord, muscleGroup = this.muscleGroup)
-}
```

Comment:
I still think its a good idea to have a mapper from the `ExerciseEntity` to this domain model class, but I need to spend some more time thinking about where it belongs. For now this extension was unused so I'm removing it 🤷‍♀

Reply:
If we expect to have a lot of these functions, maybe a utils file?